catalyst-toolbox/src/bin/cli/rewards/veterans.rs

```diff
@@ -78,47 +78,75 @@ impl VeteransRewards {
             reputation_agreement_rate_cutoffs,
             reputation_agreement_rate_modifiers,
         } = self;
-        let reviews: Vec<VeteranRankingRow> = csv::load_data_from_csv::<_, b','>(&from)?;
 
-        if rewards_agreement_rate_cutoffs.len() != rewards_agreement_rate_modifiers.len() {
-            bail!(
+        vca_rewards(
```

Comment:
any tests for the proposer logic?

Reply:
This stuff is covered by the vit-testing PR